documentation/en/setting-a-static-port.md

```diff
@@ -15,6 +15,20 @@ To change the port to `1347`:
 
 After changing the port value, restart your **daemon**.
 
+## Announce Addresses
+
+If the **swarm port** is port-forwarded from another address, it is possible to control what addresses
+are announced to the network.
+
+```sh
+[Libp2p]
+  AnnounceAddresses = ["/ip4/<public-ip>/tcp/1347"]
+```
+
+If non-empty, this array specifies the swarm addresses to announce to the network. If empty, the daemon will announce inferred swarm addresses.
+
+Similarly, it is possible to set `NoAnnounceAddresses` with an array of addresses to not announce to the network.
+
```

Comment:
```suggestion

You can check what addresses are announced with `lotus net listen`.
```

Reply:
Unfortunately, that's not really true ... `lotus net listen` shows you interfaces it is listening on, but libp2p is announcing the different address that is configured.

I wonder if it should show both? Or have a different `lotus net announced` (announcing?) command?

In go-ipfs, if I type `ipfs id`, it shows my "Announce" addresses.

Reply:
I see. I'd probably add a separate command

Reply:
(In a separate PR)